build-system/tasks/helpers.js

```diff
@@ -363,8 +363,7 @@ async function esbuildCompile(srcDir, srcFilename, destDir, options) {
             : options.outputFormat,
         banner,
         footer,
-        // For es5 builds, ensure esbuild-injected code is transpiled.
-        target: argv.esm ? 'es6' : 'es5',
+        target: 'es6',
```

Comment:
`esbuild` will yell without this change: 
```
Transforming for-of loops to the configured target environment ("es5") is not supported yet
```
Since ES6 has been widely supported since June 2017, this is probably okay. Alternatively, we can keep the babel transform on the nomodule build.

Reply:
According to `caniuse`, support is only at [95.09% of all users](https://caniuse.com/?search=es6).  Historically we've had  <1% as our target for deprecating support for a specific browser, as well as requiring an I2D